specification/storagepool/resource-manager/Microsoft.StoragePool/preview/2021-04-01-preview/examples/DiskPoolSkus_List.json

```diff
@@ -0,0 +1,31 @@
+{
+  "parameters": {
+    "api-version": "2021-04-01-preview",
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "location": "eastus"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "availabilityZones": [
+              "1",
+              "2",
+              "3"
+            ],
+            "tier": "Basic"
+          },
+          {
+            "availabilityZones": [
+              "1",
+              "2"
+            ],
+            "tier": "Standard"
+          }
+        ]
+      },
```

Comment:
This does not conform RPC.
[https://github.com/Azure/azure-resource-manager-rpc/blob/2e1f2c11ce4cfdf309f9cc8ae71575cb8fb83319/v1.0/Addendum.md#response](https://github.com/Azure/azure-resource-manager-rpc/blob/2e1f2c11ce4cfdf309f9cc8ae71575cb8fb83319/v1.0/Addendum.md#response)
[https://armwiki.azurewebsites.net/api_contracts/SKUUpdateContract.html](https://armwiki.azurewebsites.net/api_contracts/SKUUpdateContract.html)

Reply:
This is a custom api that we're implementing for the Portal team at this route: 
/subscriptions/{subscriptionId}/providers/Microsoft.StoragePool/locations/{location}/diskPoolSkus

we want to be able to communicate the availabilityZone info at this endpoint

Reply:
We decided to change the API route to the following to minimize confusion:
`/subscriptions/{subscriptionId}/providers/Microsoft.StoragePool/locations/{location}/diskPoolZones` @j5lim  please let me know if that works.

Reply:
Yes, it's fine.